playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -177,7 +177,10 @@
     SHIPYARD_IMAGE: "{{ shipyard_image }}"
     OS_PASSWORD: "{{ lookup('password', secrets_location + '/ucp_shipyard_keystone_password ' + password_opts) }}"
   register: shipyard_desc_action
-  until: shipyard_desc_action.stdout.find('Processing') < 0 and shipyard_desc_action.stdout.find('running') < 0
+  until:
+    - shipyard_desc_action.stdout.find('Processing') < 0
+    - shipyard_desc_action.stdout.find('running') < 0
+    - shipyard_desc_action.stdout.find('Error') != 0
```

Comment:
why has to be Error?

Reply:
There may be "Errors: 0" near the end of the output, so we need to limit this to only when "Error" is at the very beginning of the output.